lib/precompiled-abi-forks-util.js

```diff
@@ -36,13 +39,15 @@ const cache = new Map();
  */
 const getBridge = (rskClient, forkName = '') => {
 
-    const { orchid, wasabi100, papyrus200, iris300, hop400, hop401, fingerroot500 } = Runners.common.forks;
+    const { orchid, wasabi100, papyrus200, iris300, hop400, hop401, fingerroot500, arrowhead600 } = Runners.common.forks;
     if(cache.has(forkName)) {
         return cache.get(forkName);
     }
 
     let bridge;
     switch(forkName) {
+        case arrowhead600.name:
+            bridge = precompiledArrowhead600.bridge.build(rskClient);
```

Comment:
You are missing the `break` statement here.

Check also the default case, it's set for fingerroot

Reply:
Fixed

Reply:
What about the default case?